### PR TITLE
jsonnet: Update deps

### PR DESCRIPTION
**Pull request by LiliC**
We want to update to latest dependencies before the feature freeze. ❄️ 

cc @openshift/openshift-team-monitoring

### CODE REVIEW

File: jsonnet/thanos-querier.jsonnet [hunk 1]
```diff
@@ -29,9 +29,7 @@ local authorizationRole =
   local config = super._config,
 
   thanos+:: {
-    variables+:: {
-      image: config.imageRepos.openshiftThanos + ':' + config.versions.openshiftThanos,
-    },
+    image:: config.imageRepos.openshiftThanos + ':' + config.versions.openshiftThanos,
```
Comment (LiliC) [inline jsonnet/thanos-querier.jsonnet]: I had to update this for the generation to work, I assume this was due to some change in kube-thanos cc @metalmatze ?

File: assets/prometheus-operator-user-workload/service-monitor.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ metadata:
     app.kubernetes.io/name: prometheus-operator
     app.kubernetes.io/version: v0.33.0
   name: prometheus-operator
-  namespace: openshift-user-workload-monitoring
+  namespace: openshift-monitoring
```
Comment (LiliC) [inline assets/prometheus-operator-user-workload/service-monitor.yaml]: hmm :D why was this updated automatically incorrectly for me? 🤔

Comment (s-urbaniak) [reply assets/prometheus-operator-user-workload/service-monitor.yaml]: hmm, can you make sure that the changes from https://github.com/coreos/kube-prometheus/pull/260 is pulled in your local jsonnet vendor directory?

Comment (LiliC) [reply assets/prometheus-operator-user-workload/service-monitor.yaml]: And it seems to be taking the latest commit of kube-prometheus below.

**Comment by LiliC** [comment]
The problem was colliding versions of prometheus-operator, so lets wait until prometheus-operator is updated in kube-prometheus, after the prometheus-operator is released. :)

File: assets/grafana/deployment.yaml [hunk 3]
```diff
@@ -18,7 +18,7 @@ spec:
       containers:
       - args:
         - -config=/etc/grafana/grafana.ini
-        image: grafana/grafana:6.2.2
+        image: grafana/grafana:6.4.3
```
Comment (LiliC) [inline assets/grafana/deployment.yaml]: Did we update our grafana as well? 🤔

File: assets/prometheus-k8s/rules.yaml [hunk 4]
```diff
@@ -37,12 +37,8 @@ spec:
         )
       record: instance:node_memory_utilisation:ratio
     - expr: |
-        (
-          rate(node_vmstat_pgpgin{job="node-exporter"}[1m])
-        +
-          rate(node_vmstat_pgpgout{job="node-exporter"}[1m])
-        )
-      record: instance:node_memory_swap_io_pages:rate1m
+        rate(node_vmstat_pgmajfault{job="node-exporter"}[1m])
```
Comment (LiliC) [inline assets/prometheus-k8s/rules.yaml]: > node_vmstat_pgmajfault

I am sorry? 🤔

File: jsonnet/thanos-querier.jsonnet (hunk 1 above)
Comment (bwplotka) [reply jsonnet/thanos-querier.jsonnet]: yes I can confirm: https://github.com/thanos-io/kube-thanos/commit/aae64db90a24d1445f373f94b5d298c141ef3d41#diff-2287ff014e43baab107e2583776d124b

Thanks!

Comment (metalmatze) [reply jsonnet/thanos-querier.jsonnet]: Yup. Exactly right. :grin: :innocent:

File: assets/grafana/deployment.yaml (hunk 3 above)
Comment (s-urbaniak) [reply assets/grafana/deployment.yaml]: last bump was from june, pinned at 6.2.4, as per https://github.com/openshift/grafana/pull/18

Comment (s-urbaniak) [reply assets/grafana/deployment.yaml]: (the image is overriden anyways in here, but let's bump it next time)

Comment (s-urbaniak) [reply assets/grafana/deployment.yaml]: but good catch, I need to add this to my release checklist :+1:

Comment (paulfantom) [reply assets/grafana/deployment.yaml]: I am adding myself as an interested party to maintain downstream grafana as I am already watching what is going on in upstream repository. With that I will take care of bumping our downstream fork periodically.

**Comment by LiliC** [comment]
Screenshot 2019-11-01 at 20 16 41
Okay so all network dashboards are broken.  Will exclude those from this PR.

**Comment by metalmatze** [comment]
those network dashboards were exactly the reason I asked :upside_down_face:

File: assets/grafana/deployment.yaml (hunk 3 above)
Comment (bwplotka) [reply assets/grafana/deployment.yaml]: Are you planning to sneakly change to Spartana? @paulfantom =D

Comment (paulfantom) [reply assets/grafana/deployment.yaml]: Maybe :stuck_out_tongue:

**Pull request merged**